internal/provider/build.go

```diff
@@ -26,13 +27,6 @@ import (
 func fromImageData(ic types.ImageConfiguration, popts ProviderOpts, wd string) (*build.Context, error) {
 	ic.Contents.Packages = sets.List(sets.New(ic.Contents.Packages...).Insert(popts.packages...))
 
-	// Normalize the architecture by calling ParseArchitecture.  This is
-	// something sublte that `apko` gets for free because it only accepts yaml
-	// and the yaml parsing normalizes things.
-	for i, arch := range ic.Archs {
-		ic.Archs[i] = types.ParseArchitecture(arch.String())
-	}
-
 	bc, err := build.New(wd,
 		build.WithImageConfiguration(ic),
```

Comment:
I don't get why we move the above down, since it's used here.

Reply:
Yeah good call. Lemme unwind that.

Reply:
There's a race in `fromImageData` being called concurrently, which was what I was trying to avoid. I'll make that change separately with some more thought, later.